core/dbt/tests/fixtures/project.py

```diff
@@ -62,37 +62,55 @@ def test_data_dir(request):
     return os.path.join(request.fspath.dirname, "data")
 
 
+@pytest.fixture(scope="session")
+def dbt_profile_target(request):
+    adapter_type = request.config.getoption("--adapter")
+    if adapter_type == "redshift":
+        target = redshift_target()
```

Comment:
Yeah, this is just for development convenience to spot check running the adapter tests inside dbt-core. If it's too confusing or complicated we can remove it.

Reply:
Oh, I see. Instead of using the quote_policy that's constructed there they went back to using the default_quote_policy. Tbh, I'm not exactly sure why we're developing that non-default quote policy there. Will have to investigate more, or at least document it.

Reply:
Commented in wrong place...

Reply:
After thinking about this, I think that it's a bit confusing and misleading to have multiple profile targets, so I've removed the option and the redshift target. It was a minor convenience and not worth the possible confusion.